src/dataflow/src/sink/kafka.rs

```diff
@@ -129,6 +131,70 @@ impl ProducerContext for SinkProducerContext {
     }
 }
 
+#[derive(Debug)]
+pub enum SinkConsistencyState {
+    Begin,
+    End,
```

Comment:
So do we expect consumers of the sink's consistency topic to "close out" timestamps based on messages with `End`?

Reply:
yup!